src/main/java/com/salesforce/dataloader/process/DataLoaderRunner.java

```diff
@@ -34,26 +34,37 @@
 import com.salesforce.dataloader.exception.ControllerInitializationException;
 import com.salesforce.dataloader.ui.UIUtils;
 
+import java.util.Map;
+
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Shell;
 
 public class DataLoaderRunner {
 
     public static final String UI = "ui";
```

Comment:
Are these variables used outside of this class? Can we narrow the scope if we can?

Reply:
good catch. I have made the change and updated the PR.